### PR TITLE
Ghostnet 연습 번역 추가

### PR DESCRIPTION
**Pull request by KyubumShin**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

Ghostnet 번역을 추가하였습니다

### CODE REVIEW

File: pytorch_vision_ghostnet.md [hunk 1]
```diff
@@ -81,20 +81,21 @@ for i in range(top5_prob.size(0)):
     print(categories[top5_catid[i]], top5_prob[i].item())
 ```
 
-### Model Description
+### 모델 설명
 
-The GhostNet architecture is based on an Ghost module structure which generate more features from cheap operations. Based on a set of intrinsic feature maps, a series of cheap operations are applied to generate many ghost feature maps that could fully reveal information underlying intrinsic features. Experiments conducted on benchmarks demonstrate that the superiority of GhostNet in terms of speed and accuracy tradeoff.
+고스트넷 아키텍처는 다양한 특징 맵을 효율적인 연산으로 생성하는 고스트 모듈 구조로 이루어집니다. 고유한 특징 맵을 기반으로 한 효율적인 연산으로 추론에 기반이 되는 고유 특징들을 충분하게 드러내는 고스트 특징 맵을 생성합니다. 벤치마크에서 수행된 실험을 통해 속도와 정확도의 상충 관계에 관한 고스트넷의 우수성을 보여줍니다.
```
Comment (euroversedev) [inline pytorch_vision_ghostnet.md:86]: 개인적으로 'feature maps' 대비 '특징 맵'이라는 단어가 부자연스럽다는 느낌이 듭니다 !  : <

**Review by euroversedev — COMMENTED** [review]
꼬투리를 잡아봤습니다 하하

File: pytorch_vision_ghostnet.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ model = torch.hub.load('huawei-noah/ghostnet', 'ghostnet_1x', pretrained=True)
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+모든 사전 학습된 모델들은 입력 이미지가 동일한 방식으로 정규화 되는 것을 요구합니다. 
+다시 말해 `H`와 `W`가 적어도 `224`이고, `(3 x H x W)`의 shape를 가지는 3채널 RGB 이미지들의 미니배치를 말합니다.
```
Comment (spongebob03) [inline pytorch_vision_ghostnet.md:27]: 설명부분과 예시 부분으로 나뉘어져서 자연스럽게 이해하기 좋았습니다!
